checkov/terraform/graph_builder/variable_rendering/evaluate_terraform.py

```diff
@@ -232,16 +232,18 @@ def _handle_literal(input_str: str) -> str:
     try:
         e = ast.literal_eval(input_str)
         if isinstance(e, list) and len(e) == 1:
-            return e[0]
+            return cast("str", e[0])
```

Comment:
why do we need this cast? I think maybe it's better to cast `e` itself.
also, why isn't it `str | bool | int`?

Reply:
actually it should be `Any`